content/2-2.md

```diff
@@ -1,44 +1,46 @@
 ---
-title: "[Draft] Module 2: Navigational Menus"
-permalink: /curricula/developing-accessible-content/navigational-menus/
-ref: /curricula/developing-accessible-content/navigational-menus/
+title: "[Draft] Module 2: Menus"
+permalink: /curricula/developer-modules/menus/
+ref: /curricula/developer-modules/menus/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-2.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/structure-and-semantics/
-  next: /curricula/developing-accessible-content/images-and-graphics/
+  previous: /curricula/developer-modules/page-structure/
+  next: /curricula/developer-modules/images/
 ---
  
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
 
-* explain use of proper coding to provide accessible structures, labeling, and functionality of menus
-* demonstrate how people with disabilities use structure, labels, markup, and styling of menus to navigate
+* demonstrate how people with disabilities use structures, labels, and styling to interact with menus
+* explain coding techniques to provide accessible structures, labels, and functionality of menus
 
 ## Learning Outcomes for Module
 
 Students should be able to:
 
-* identify menus using headings or the HTML element `nav`
-* code menu structures using the HTML elements `ul` and `li`,
-* code menu items using the HTML element `a` and the attribute `href`
-* label menu items using link texts or images with their corresponding alternative text
-* convey the hierarchy of menu items using nested lists
-* mark up different states of menu items, such as hover, focus, current, or active
+* explain strategies that people with disabilities use to interact with menus
+* mark up labels to identify menus provided on a page
```

Comment:
For consistency, should this be "code" rather than "mark up"?

Reply:
Good point. It affects the whole resource actually, though here it can get specially confusing.

We were trying to make a distinction between:

* mark up -- provide semantics to presentation that might be visually apparent
* code -- may involve more of a set of programming skills

1. Do you thinkg that distinction is worthy?
    If so, I will take a general pass at the whole subset of "code" versus "mark up" and aupdate accordingly.

2. Do you think consistency is more important here as those differences may be very suttel and sometimes unnoticeable?
    If so, what verb do you think better conveys the message here? "Mark up", "code", other?

Reply:
I think that the distinction might be too subtle to deal with effectively although I do appreciate the goal.

My gut would be to go with 'code' rather than 'mark up'. I tend to try to use 'mark up' to describe the result rather than the action. Although I am probably as guilty of inconsistency!